### PR TITLE
Fix accessibility errors and alerts for broker registration views

### PR DESCRIPTION
**Pull request by bbodine1**
Ticket: https://www.pivotaltracker.com/story/show/188168765

# A brief description of the changes

Current behavior: Many accessibility errors

New behavior: No errors

1 warning remains due to PDF link. There is not way to fix this without using html to create the PDF.

### CODE REVIEW

**Review by kristinmerbach — APPROVED** [review]
LGTM

File: app/javascript/css/main.scss [hunk 1]
```diff
@@ -178,6 +178,26 @@ dl.parent.weight-n {
   }
 }
 
+.tax_credit_field_container {
+  height: 44px !important;
+}
+
+#personal_info {
```
Comment (TristanB17) [inline app/javascript/css/main.scss:185]: I think the styling for `#personal-info` is also present in `css/forms.scss` with a slight change.

Are we repeating it here for any particular reason? Not a big deal, just curious!

Comment (bbodine1) [reply app/javascript/css/main.scss:185]: This came in from a rebase on trunk. ¯\_(ツ)_/¯

**Review by TristanB17 — APPROVED** [review]
These additions are great! I'll have to be more mindful about accessibility in the future.

**Pull request merged**